### DISCUSSION TITLE
Kagi

### DISCUSSION THREAD

**Discussion by es183923** [catppuccin/catppuccin#1852] [Port Requests]
link: https://kagi.com/

**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
without an account (free or paid), it needs to use stylus
with an account, you can get theming with custom css (limited at 10k chars) everywhere except the settings pages

![image](https://user-images.githubusercontent.com/72744903/196042741-18d87fa1-0af5-45d0-b78c-adcf5837ef8d.png)

![image](https://user-images.githubusercontent.com/72744903/196042747-a48b5536-ed84-4a5c-8a7e-d493ad221e4b.png)

**Comment by sgoudham** [comment]
Closing since https://github.com/catppuccin/catppuccin/issues/2642 has been created and is in-review.
